src/components/services/Ping.vue

```diff
@@ -5,6 +5,14 @@
         {{ status }}
       </div>
     </template>
+    <template #content>
+      <p class="title is-4">{{ item.name }}</p>
+      <p class="subtitle is-6">
+      <template v-if="status === 'online' && item.showRtt">
+        {{ rtt }} ms
+      </template>
```

Comment:
It would be great to still have the subtitle if `showRtt` is set to false, could you add it? 
Also maybe, if `showRtt` is set to true, and service is not online, it could say "Unknown RTT" or something like that?

Reply:
Sure thing! I’ve updated it.